### PR TITLE
Feature 153 add image restaurant card voznyarskyy

### PR DESCRIPTION
**Pull request by VozNick**
!!! MIGRATIONS INCLUDED

-add posibility to upload restaurant photos (one or many)
-valitation works (max selected files = 20, max file size = 5 mb, types = png/jpg/jpeg = will all be saved as jpg)
-photos shown on rest card (if no photo - default)
-add gallery to main page restaurant

### CODE REVIEW

**Review by vitaliimak — COMMENTED** [review]
Your task in "In progress" and it doesn't have estimate and description. I think, you should add POST method to add images to restaurants

**Comment by vitaliimak** [comment]
@VozNick could you add new reviewers and remove old?

**Comment by VozNick** [comment]
not working from mobile, will come home in hour and it from PC

**Comment by vitaliimak** [comment]
@VozNick no problem, don't rush
And you can use official github app for mobile phone https://play.google.com/store/apps/details?id=com.github.android
Screenshot_20200507-101627_GitHub

**Comment by VozNick** [comment]
official, 
but I just updated, and now I can see edit button ))

**Review by Andrii35 — COMMENTED** [review]
I think "Or Drag it Here" should be in the center and don't touch other text or button.

addImage

File: server/src/main/java/com/lunchtime/mapper/OrderMapper.java [hunk 1]
```diff
@@ -38,7 +38,8 @@ public Order fromDtoToOrder(OrderDto orderDto) throws IOException {
         order.setStartTime(orderDto.getStartTime());
         order.setFinishTime(orderDto.getFinishTime());
         order.setDescription(orderDto.getDescription());
-        order.setVisitors(orderDto.getVisitors());;
+        order.setVisitors(orderDto.getVisitors());
+        ;
```
Comment (narekacy) [inline server/src/main/java/com/lunchtime/mapper/OrderMapper.java:45]: Could you remove line 42?

**Review by narekacy — APPROVED** [review]

**Review by Andrii35 — APPROVED** [review]

**Review by vitaliimak — APPROVED** [review]

**Pull request merged**